tatau_core/settings/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 from dotenv import load_dotenv, find_dotenv
 
-dotenv_path = os.getenv('DOTENV_PATH', '.env')
+dotenv_path = os.getenv('DOTENV_PATH', '.env.dev')
```

Comment:
I think default value should be for production. Something like that
```suggestion
dotenv_path = os.getenv('DOTENV_PATH', '.env.prod')
```